presentation/src/main/java/me/androidbox/presentation/agenda/viewmodel/AgendaViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package me.androidbox.presentation.agenda.viewmodel
+
+import androidx.lifecycle.ViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import me.androidbox.presentation.agenda.screen.AgendaScreenEvent
+import me.androidbox.presentation.agenda.screen.AgendaScreenState
+import javax.inject.Inject
+
+@HiltViewModel
+class AgendaViewModel @Inject constructor() : ViewModel() {
+
+    private val agendaScreenMutableState = MutableStateFlow(AgendaScreenState())
+    val agendaScreenState = agendaScreenMutableState.asStateFlow()
```

Comment:
Any reason you don't stick to the general convention of just prepending an underscore to the mutable flow? (`_agendaScreenState`)
I haven't seen it be done differently anywhere

Reply:
@philipplackner Updated to having a underscore for the postfix `_agendaScreenState`